### PR TITLE
Django Tutorial Part 6: Generic list and detail views

### PR DESCRIPTION
**Pull request by huutuan2308**

### CODE REVIEW

**Comment by huutuan2308** [comment]
ready

**Comment by huutuan2308** [comment]
ready

File: catalog/templates/base_generic.html [hunk 1]
```diff
@@ -17,6 +17,33 @@
       {% block sidebar %}
         <ul class="sidebar-nav">
           <li><a href="{% url 'index' %}">Home</a></li>
+<<<<<<< HEAD
```
Comment (thaodtd-0665) [inline catalog/templates/base_generic.html]: Conflict code còn nè em ơi

**Comment by huutuan2308** [comment]
ready

**Review by thaodtd-0665 — APPROVED** [review]

**Comment by thaodtd-0665** [comment]
merged

**Pull request merged**